library/system/test/commented_config_file_test.rb

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/rspec
+#
+# Unit test for CommentedConfigFile
+#
+# (c) 2017 Stefan Hundhammer <Stefan.Hundhammer@gmx.de>
+#     Donated to the YaST project
+#
+# Original project: https://github.com/shundhammer/ruby-commented-config-file
+#
+# License: GPL V2
+#
+
+require_relative "test_helper"
+require "yast2/commented_config_file"
+
+describe CommentedConfigFile do
+  # rubocop:disable Lint/AmbiguousRegexpLiteral
+
+  context "when created empty" do
+    subject { described_class.new }
```

Comment:
IIRC `subject { described_class.new }` is the default, this line can be omitted. (See [Implicitly defined subject](https://relishapp.com/rspec/rspec-core/v/3-6/docs/subject/implicitly-defined-subject) documentation.)